src/h5web/vis-packs/core/containers/RgbVisContainer.tsx

```diff
@@ -19,6 +20,12 @@ function RgbVisContainer(props: VisContainerProps) {
   assertNumDims(entity, 3);
   assertNumericType(entity);
 
+  const subclassAttr = getAttributeValue(entity, 'IMAGE_SUBCLASS');
+  if (typeof subclassAttr !== 'string' || subclassAttr !== 'IMAGE_TRUECOLOR') {
+    // eslint-disable-next-line no-console
+    console.warn('RGB Vis only supports IMAGE_TRUECOLOR.');
```

Comment:
I was more suggesting throwing if the subclass attribute _is_ provided AND _is not_ `IMAGE_TRUECOLOR`. The idea is to fail early if we know for sure the image is not truecolor. If the subclass attribute is not provided, we can't tell, so we give the image the benefit of the doubt.

Reply:
Sorry, I misunderstood. Your proposition makes sense :+1: